assets/scripts/gallery/fetch_street.js

```diff
@@ -42,7 +42,8 @@ export function fetchGalleryStreet (streetId) {
     .catch(errorReceiveGalleryStreet)
 }
 
-function errorReceiveGalleryStreet () {
+function errorReceiveGalleryStreet (err) {
+  console.log(err)
```

Comment:
Doesn't need to be in this commit, but we should consider displaying some kind of alert when an ajax request fails

Reply:
Cool, I'll file this here: https://github.com/streetmix/streetmix/issues/802